livox_lidar_publisher:
  ros__parameters:
    xfer_format   : 0    # 0-Pointcloud2(PointXYZRTL), 1-customized pointcloud format
    multi_topic   : 0    # 0-All LiDARs share the same topic, 1-One LiDAR one topic
    data_src      : 0    # 0-lidar, others-Invalid data src
    publish_freq  : 10.0 # freqency of publish, 5.0, 10.0, 20.0, 50.0, etc.
    output_type   : 0
    frame_id      : 'livox_frame'
    lvx_file_path : '/home/livox/livox_test.lvx'
    cmdline_bd_code : 'livox0000000001'

livox_repub_node:
  ros__parameters:
    point_cloud_type  : 0
    combine_cloud_size: 2
    num_skip_cloud    : 2
    publish_rate      : 2.0
